### PR TITLE
PPTP-1968

### PR DESCRIPTION
**Pull request by C-Thomp**

### CODE REVIEW

File: app/views/ManufacturedPlasticPackagingView.scala.html [hunk 1]
```diff
@@ -15,19 +15,29 @@
...
     govukRadios: GovukRadios,
     govukButton: GovukButton,
-    sectionHeader: ReturnQuarterHeading
+    sectionHeader: ReturnQuarterHeading,
+    paragraphBody: paragraphBody
 )
 
 @(form: Form[_], mode: Mode, taxReturn: TaxReturnObligation)(implicit request: Request[_], messages: Messages)
 
+@guidanceLink = {<a href="https://www.gov.uk/guidance/decide-if-you-need-to-register-for-plastic-packaging-tax" class="govuk-link" id="govuk-guidance-link">@messages("manufacturedPlasticPackaging.paragraph.link")</a>}
```
[the remark this answers is not in the document]
Comment (C-Thomp) [reply app/views/ManufacturedPlasticPackagingView.scala.html]: ah, fixed 👍

File: test-utils/generators/Generators.scala [hunk 2]
```diff
@@ -80,6 +80,11 @@ trait Generators
       .suchThat(!_.isValidInt)
       .map(_.formatted("%f"))
 
+  def intsInRangeWithCommas(min: Long, max: Long): Gen[String] = {
```
Comment (ajchilts) [inline test-utils/generators/Generators.scala:83]: Delete this. Not needed and we should not upset the Int behaviour.

**Review by ajchilts — APPROVED** [review]

File: test-utils/generators/Generators.scala (hunk 2 above)
Comment (C-Thomp) [reply test-utils/generators/Generators.scala]: done

File: app/views/ManufacturedPlasticPackagingWeightView.scala.html [hunk 3]
```diff
@@ -43,11 +60,15 @@
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = LabelViewModel(messages("manufacturedPlasticPackagingWeight.heading")).asPageHeading()
+                label = LabelViewModel(messages("manufacturedPlasticPackagingWeight.heading")).asPageHeading(size = LabelSize.Large)
```
[the remark this answers is not in the document]
Comment (C-Thomp) [reply app/views/ManufacturedPlasticPackagingWeightView.scala.html:63]: separate, what we can do if change the default on the models for Label and Legend

**Pull request merged**